packages/ui/src/ui-component/lists/FlowListView.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types'
+import { Box, Grid, Skeleton, Typography } from '@mui/material'
+import ItemCard from '@/ui-component/cards/ItemCard'
+
+const FlowListView = ({ data, images, nodeTypes, isLoading, updateFlowsApi, setError, type, onItemClick }) => {
```

Comment:
What is the point of "isLoading" here if we aren't loading anything in this component and not updating the value of it from within the component?   Could we not just check if the data props exists or not?

Reply:
to prevent flashing I believe so it has the skeleton. Its a pattern that they used so continued with it